src/App.tsx

```diff
@@ -1,8 +1,68 @@
-import React from 'react';
+import React, { FC, useState } from 'react';
 import './App.css';
+import { getPreparedList } from './Components/preparedList/preparedList';
+import { ListsState } from './Components/interfaces/interfaces';
+import { PostsLists } from './Components/PostsList/PostsList';
+import { SearchBar } from './Components/SearchBar/SearchBar';
+import { Button } from './Components/LoadButton/LoadButton';
 
-const App = () => (
-  <h1>Dynamic list of posts</h1>
-);
+const App: FC = () => {
+  const [postList, setPostsList] = useState<ListsState>({
+    initialList: [],
+    filteredList: [],
+  });
+  const [isLoading, setLoading] = useState<string>('Load');
```

Comment:
примитивные типы не указываются + это не булиан
```suggestion
  const [loadMessage, setLoadMessage] = useState('Load');
```